index.js

```diff
@@ -69,12 +70,32 @@ function ProcessReporter(options) {
         self.prefix = self.prefix + '.';
     }
 
+    self.handleEnabled = typeof options.handleEnabled === 'boolean' ?
+        options.handleEnabled :
+        true;
```

Comment:
```javascript
self.handleEnabled = options.handleEnabled || true;
assert(typeof self.handleEnabled === 'boolean', 'expected options.handleEnabled to be boolean');
```

It defaults to true, otherwise takes it from options, and we assert that what we got was a boolean.